service_mesh/service_mesh_install/preparing-ossm-installation.adoc

```diff
@@ -9,17 +9,20 @@ Before you can install {ProductName}, review the installation activities, ensure
 .Prerequisites
 
 * Possess an active {product-title} subscription on your Red Hat account. If you do not have a subscription, contact your sales representative for more information.
+* Review the xref:../../architecture/architecture-installation.adoc#installation-overview_architecture-installation[{product-title} 4.1 overview].
 * Install {product-title} 4.1.
```

Comment:
Do we need to add details for special configuration required by Elasticsearch (used by Jaeger)?  My understanding is that the default machine configuration for nodes won't support deploying a pod that requires 16Gi of memory.  I don't know if those details belong here or with the Jaeger portion of the documentation.

Reply:
I think this is being covered in the Jaeger docs

Reply:
@JStickler Do you know where this will be documented?

Reply:
The updated Jaeger Configuration is in the ossm-configuring-jaeger.adoc file.